CMakeLists.txt

```diff
@@ -42,7 +52,7 @@ if(MDNSD_BUILD_FUZZING OR MDNSD_BUILD_OSS_FUZZ)
 endif()
 
 add_executable(mdnsd mdnsd.c)
-target_link_libraries(mdnsd libmdnsd)
+target_link_libraries(mdnsd PRIVATE libmdnsd)
```

Comment:
You forgot mquery below.
And can't the 3 lines below here be removed?
If not, then you should modify `  target_link_libraries(libmdnsd PRIVATE wsock32 ws2_32)` in `libmdnsd/CMakeLIsts.txt` to
```
  target_link_libraries(libmdnsd PUBLIC wsock32 ws2_32)
```